.tree-sitter-lint.yml

```diff
@@ -8,5 +8,8 @@ rules:
     level: error
   no-lazy-static:
     level: error
-  replace-foo-with-bar/replace-foo-with-bar:
+  replace-foo-with/replace-foo-with-bar:
     level: error
+  replace-foo-with/replace-foo-with-something:
+    level: error
+    options: whee
```

Comment:
So per previous PR comment it seems like it would still probably be nice to support a more concise ESLint-config-style way of specifying the options, eg here:
```
replace-foo-with/replace-foo-with-something: [error, whee]
```
That seems doable currently with just I assume some custom `Deserialize`'ing

Where it gets interesting is that in ESLint it really models the options as a list (and so if you say eg `some-rule: [error, option1, option2]` the rule gets passed `[option1, option2]` as its options)

I think that's clunky sometimes because often that's going to be a "trivially single-item list" (eg if you model the options as a single `key: value`-style options object)

So I don't know, I like that the starting point here is not modeling them as a list but might be nice to try and smoothly support that style for rules that want to have their options passed that way?